Assets/BossRoom/Scripts/Client/UI/UISettingsCanvas.cs

```diff
@@ -11,13 +11,17 @@ public class UISettingsCanvas : MonoBehaviour
         [SerializeField]
         private GameObject m_SettingsPanelRoot;
 
+        [SerializeField]
+        private GameObject m_QuitPanelRoot;
+
         void Awake()
         {
             // the settings canvas should exist in all scenes!
             DontDestroyOnLoad(gameObject);
 
             // hide the settings window at startup (this is just to handle the common case where an artist forgets to disable the window in the prefab)
             m_SettingsPanelRoot.SetActive(false);
+            m_QuitPanelRoot.SetActive(false);
```

Comment:
Another case to consider is disabling the canvas when changing scenes, if it has been enabled. I only mention this since the context of the window can change.